charts/dapr/charts/dapr_operator/templates/dapr_operator_deployment.yaml

```diff
@@ -175,6 +175,20 @@ spec:
         - "{{ .Values.global.prometheus.port }}"
 {{- else }}
         - "--enable-metrics=false"
+{{- end }}
+{{- with .Values.global.issuerFilenames }}
+{{- if hasKey . "ca" }}
```

Comment:
Any reason why this is doing a dynamic lookup on a key rather than having a pre-defined struct? IMHO it would be simpler (for users) if the keys were defined in values.yaml explicitly

Reply:
agree, I will put a comment  with example to explain in values